sdk/keyvault/azure-keyvault-securitydomain/tests/jwe.py

```diff
@@ -0,0 +1,352 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import base64
+import hashlib
+import hmac
+import json
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes, padding
+from cryptography.hazmat.primitives.asymmetric import padding as asymmetric_padding
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+
+from utils import Utils
+
```

Comment:
Would it be worth adding a header comment to describe and link to where this module is copied from and how it should be maintained? Or is that not really necessary, as the source we copied from isn't really a source of truth to be kept in sync with?

Reply:
I don't think it would hurt to add the context of where this came from -- will update